charts/brigade-project/values.yaml

```diff
@@ -56,8 +56,12 @@ secrets:
 # you know what you are doing.
 # namespace: "default"
 
-# OPTIONAL: vcsSidecar is the image that fetches a repo from a VCS
-# vcsSidecar: "deis/git-sidecar:latest"
+# RECOMMENDED: vcsSidecar is the image that fetches a repo from a VCS
+# The default sidecar uses Git to fetch a copy of the project. Commenting this
+# out will prevent the worker from using a sidecar. This may improve performance
+# very slightly, but will break some gateways or cause the default script to
+# be used.
```

Comment:
Appreciate covering relationships with both the gateway and the default script.
I believe that this definitely helps us understand the gotachas without consuming too much time!